src/main/java/com/example/demo/model/User.java

```diff
@@ -30,5 +46,53 @@ public class User {
     @Column(nullable = false)
     private String lastName;
 
+    @ManyToMany(fetch = FetchType.EAGER, cascade = { CascadeType.ALL })
```

Comment:
1) Cascading for ManyToMany relationships, especially REMOVE, is a bad idea as it can cause bugs.
2) Don't use fetchType = EAGER, instead use `@EntityGraph` or `@Query + JOIN FETCH` in the repository method.
```suggestion
    @ManyToMany
```